Sources/AblyChat/Room.swift

```diff
@@ -106,26 +108,65 @@ internal actor DefaultRoom<LifecycleManagerFactory: RoomLifecycleManagerFactory>
             roomID: roomID,
             logger: logger
         ) : nil
+
+        _presence = options.presence != nil ? await DefaultPresence(
+            featureChannel: featureChannels[.presence]!,
```

Comment:
How linting tool allowed this?

Reply:
the force unwrap? We have a ticket/comment above to resolve this later. Lint didn't pick this up though